sonarr/qualitydefinition.go

```diff
@@ -13,12 +13,13 @@ import (
 
 // QualityDefinition is the /api/v3/qualitydefinition endpoint.
 type QualityDefinition struct {
-	ID        int64            `json:"id,omitempty"`
-	Weight    int64            `json:"weight"`
-	MinSize   float64          `json:"minSize"`
-	MaxSize   float64          `json:"maxSize"`
-	Title     string           `json:"title"`
-	Qualities []*starr.Quality `json:"items"`
+	ID       int64              `json:"id,omitempty"`
+	Weight   int64              `json:"weight"` // This should not be changed.
+	MinSize  float64            `json:"minSize"`
+	MaxSize  float64            `json:"maxSize"`
+	PrefSize float64            `json:"preferredSize"` // v4 only.
```

Comment:
if it's with v4 only, should we add `omitempty` as well to avoid filling it up when not used?

Reply:
Omit-empty will make json not print `0`, and `0` is valid, so that will break v4. As-is, v3 ignores the extra fields, so this doesn't seem to break anything.

Reply:
I've been mulling real hard about how to deal with the v4 changes. So far I've been able to "sneak them in." Meaning, they're just extra fields, or old fields that are now unused. Everything seems to work.. I opened #58 to track v4 changes/problems.

Reply:
ok, if you tried and doesn't cause any issue, it's fine!